cpp/src/io/parquet/experimental/hybrid_scan_impl.hpp

```diff
@@ -304,25 +327,6 @@ class hybrid_scan_reader_impl {
   void setup_next_pass(std::vector<rmm::device_buffer> column_chunk_buffers,
                        parquet_reader_options const& options);
 
-  /**
```

Comment:
All these member functions now no longer needed as we can reuse their base versions